update: ownership permissions clarity about client-server (#1477)
Unity-Technologies/com.unity.multiplayer.docs: docs/basics/ownership.md

![Distributed authority new client](/img/distributed-authority-new-client.jpg)

### Ownership permission settings (Distributed Authority Only)

> The following ownership permission settings, defined by `NetworkObject.OwnershipStatus`, will only take effect when using a distributed authority network topology:

Reviewer: ```suggestion
The following ownership permission settings, defined by `NetworkObject.OwnershipStatus`, only take effect when using a distributed authority network topology:
```

You can also use `NetworkObject.SetOwnershipLock` to lock and unlock the permission settings of a NetworkObject for a period of time, preventing ownership changes on a temporary basis.

::: warning

> The ownership permissions will become visible when the Multiplayer Services SDK package is installed and you are inspecting a `NetworkObject` within the editor. The ownership permissions will have no impact when using a client-server network topology since the server always has authority. In order for the ownership permissions to be utilized, you must be using a distributed authority network topology.

Reviewer: ```suggestion
The ownership permissions are only visible when the Multiplayer Services SDK package is installed and you're inspecting a `NetworkObject` within the editor. Ownership permissions have no impact when using a client-server network topology, since the server always has authority. For ownership permissions to be used, you must be using a distributed authority network topology.
```
